examples/morpheusvm/actions/common.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
+
+const (
+	// IDs explicitly listed. Registry will avoid usage of duplicated IDs by returning an error when occurs
```

Comment:
nit (for all):

```
// Note: Registry will error during initialization if a duplicate ID is assigned. We explicitly assign IDs to avoid accidental remapping.
```

Reply:
You can also put this just above `const`